src/main/java/com/google/sps/data/ScheduledInterviewDao.java

```diff
@@ -29,11 +28,12 @@
   // Returns a list of the ScheduledInterview objects that the user participates in.
   public List<ScheduledInterview> getForPerson(String email);
 
-  // Creates a ScheduledInterview entity.
-  public Entity create(ScheduledInterview scheduledInterview);
+  // Creates a ScheduledInterview entity and stores it in Datastore.
+  public void create(ScheduledInterview scheduledInterview);
```

Comment:
Can you change create to put to match Datastore syntax?
Also less confusing for the Autovalue create()